src/frontends/common/src/graph_iterator.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/graph_iterator.hpp"
+
+using namespace ov::frontend::tensorflow;
+
+std::map<std::string, std::string> GraphIterator::get_input_names_map() const {
+    return {};
+}
+
+std::map<std::string, std::string> GraphIterator::get_output_names_map() const {
+    return {};
```

Comment:
why are we using tensorflow namespace in common frontend. Looks weird.

Reply:
you have approved these changes in PR https://github.com/openvinotoolkit/openvino/pull/16355

If you want to drop namespace in common part, please, perform it separately.

Reply:
It originally comes from this namespace, but place under FE general includes.

Reply:
> you have approved these changes in PR #16355
> 
> If you want to drop namespace in common part, please, perform it separately.

I can't find that we merged use of tf namespace there.

Reply:
> It originally comes from this namespace, but placed under FE general includes.

Yes, but in common part we never mention any TF FE specifics. Now it introduce confusion and cyclic dependencies: common FE- >TF FE->common FE.

Reply:
CVS-161084